src/pages/generate/components/result/ResultPage.tsx

```
@@ -2,54 +2,34 @@ import { useState } from 'react';
 import Modal from '@/shared/components/overlay/modal/Modal';
-import Loading from '@/shared/components/loading/Loading';
 
 const ResultPage = () => {
   const location = useLocation();
-  const [searchParams] = useSearchParams();
 
   // Hook들을 최상단에 배치
   const [selected, setSelected] = useState<'like' | 'dislike' | null>(null);
 
   // 1차: location.state에서 데이터 가져오기 (정상적인 플로우)
   let result = (location.state as { result?: GenerateImageData })?.result;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)